src/sql/workbench/api/common/sqlExtHostTypes.ts

```diff
@@ -94,3 +94,21 @@ export interface IComponentEventArgs {
 	eventType: ComponentEventType;
 	args: any;
 }
+
+export interface IModelViewDialogDetails {
+	title: string;
+	content: string | number[];
+	okTitle: string;
+	cancelTitle: string;
```

Comment:
Do we support the case where there's only an OK button? E.g. some cases where things will always succeed?